training/src/anemoi/training/config/model/graphtransformer_ens.yaml

```diff
@@ -68,12 +23,26 @@ processor:
   num_heads: 16 # GraphTransformer or Transformer only
   qk_norm: True # Transformer and GraphTransformer only
   cpu_offload: ${model.cpu_offload}
-
+  layer_kernels:
+    LayerNorm:
+      _target_: anemoi.models.layers.normalization.ConditionalLayerNorm
+      normalized_shape: ${model.num_channels}
+      condition_shape: ${model.noise_injector.noise_channels_dim}
+      w_one_bias_zero_init: True
+      autocast: false
+      #Any arguments to your chosen function go here
+    Linear:
+      _target_: torch.nn.Linear
+    QueryNorm:
+      _target_: anemoi.models.layers.normalization.AutocastLayerNorm
+      bias: False
+    KeyNorm:
+      _target_: anemoi.models.layers.normalization.AutocastLayerNorm
+      bias: False
 encoder:
   _target_: anemoi.models.layers.mapper.GraphTransformerForwardMapper
   trainable_size: ${model.trainable_parameters.data2hidden}
   sub_graph_edge_attributes: ${model.attributes.edges}
-  activation: ${model.activation}
```

Comment:
Similar to my previous question, wondering if we should state the layer kernel explicitly in encoder and decoder as done for the `gnn.yaml`

( and same question for the graphtransformer_ens and transformer files)